resources/85-tessel.rules

```diff
@@ -0,0 +1 @@
++ATTRS{idVendor}=="1209", ATTRS{idProduct}=="7551", ENV{ID_MM_DEVICE_IGNORE}="1", MODE="0666"
```

Comment:
No `+` at the beginning of the line. (Was this copied from a diff?)

This should have a different name, or contain both T1 and T2 rules. Otherwise, if the user has the T1 CLI installed, it will overwrite the file with the T1 rules.